### ISSUE TITLE
[Crash] Wadanohara "Seabather A watches things unfold.."

### ISSUE THREAD

**Issue by Kikasuru** [EasyRPG/Player#1808]
(post 0.6.1 regression)

#### Name of the game:
Wadanohara and the Great Blue Sea (vgperson Translation v1.04)

#### Player platform:
Android (Tested on 2 systems.)

#### Attach files (as a .zip archive or link them)
 - A savegame next to the problem (if it is not at the beginning of the game)
[Save Game](https://drive.google.com/file/d/1jy9Eh97W27_juLXML12GxeAIVESddUzO/view?usp=drivesdk)
 - The `easyrpg_log.txt` log file (you can find it in the game directory)
[Log](https://drive.google.com/file/d/1VsHL7kVg9CivL8sRU4nJQViwLVfyVOr_/view?usp=drivesdk)

#### Describe the issue in detail and how to reproduce it:
When fighting the first enemy in Wadanohara, the game will unexpectedly crash after the "Seabather A watches things unfold.." text appears. I wish I had more information to give, but I'm in the complete blank to what exactly happened.

**Comment by Ghabry** [comment]
Thanks for the bug report.

@Kikasuru Did you install the app update published today (0.6.0-319)?

We were expecting some crashes in the new release but that it crashes in the first battle is a bit emberassing xD.

**Comment by fmatthew5876** [comment]
Fixed by #1809. Tested it in Wadanohara with the provided save game.

**Comment by Ghabry** [comment]
Android stacktrace via dev console:

```
  #00  pc 00000000001c5f38  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZNK12Game_Battler6IsDeadEv+3)
  #01  pc 00000000001beb21  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZN20Game_BattleAlgorithm13AlgorithmBase5ApplyEv+160)
  #02  pc 00000000001c1177  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZN20Game_BattleAlgorithm5Skill5ApplyEv+66)
  #03  pc 0000000000219969  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZN19Scene_Battle_Rpg2k319ProcessBattleActionEPN20Game_BattleAlgorithm13AlgorithmBaseE+512)
  #04  pc 00000000002195d5  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZN19Scene_Battle_Rpg2k314ProcessActionsEv+124)
  #05  pc 0000000000215559  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZN12Scene_Battle6UpdateEv+272)
  #06  pc 0000000000217fdf  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZN19Scene_Battle_Rpg2k36UpdateEv+254)
  #07  pc 00000000001f952d  /data/app/org.easyrpg.player/lib/arm/libmain.so (_ZN6Player6UpdateEb+516)
```

This Wadanohara crash: It looks slightly different (Skill::Apply is not on the stack) but the cause is the same imo:

```
Game_Battler::HasState game_battler.cpp:43
Game_Battler::IsDead game_battler.cpp:88
Game_BattleAlgorithm::AlgorithmBase::Apply game_battlealgorithm.cpp:631
Scene_Battle_Rpg2k::ProcessActionApply scene_battle_rpg2k.cpp:633
Scene_Battle_Rpg2k::ProcessBattleAction scene_battle_rpg2k.cpp:442
Scene_Battle_Rpg2k::ProcessNextAction scene_battle_rpg2k.cpp:393
Scene_Battle_Rpg2k::ProcessActionExecute scene_battle_rpg2k.cpp:621
Scene_Battle_Rpg2k::ProcessBattleAction scene_battle_rpg2k.cpp:438
Scene_Battle_Rpg2k::ProcessActions scene_battle_rpg2k.cpp:322
Scene_Battle::Update scene_battle.cpp:173
Scene_Battle_Rpg2k::Update scene_battle_rpg2k.cpp:56
Player::Update player.cpp:309
```

**Comment by Kikasuru** [comment]
> Thanks for the bug report.
> 
> @Kikasuru Did you install the app update published today (0.6.0-319)?
> 
> We were expecting some crashes in the new release but that it crashes in the first battle is a bit emberassing xD.

Sure did, and that is quite embarrassing..

**Comment by Ghabry** [comment]
@Kikasuru 
We published a new APK, should appear in the store soon.

Based on our crash logs this is the only crash reported by now, so happy gaming now I guess.
